docker/Dockerfile

```diff
@@ -26,6 +27,7 @@ RUN wget -O /tmp/install-tl-unx.tar.gz http://mirror.ctan.org/systems/texlive/tl
         fontspec frcursive fundus-calligra gnu-freefont jknapltx latex-bin \
         mathastext microtype ms physics preview ragged2e relsize rsfs \
         setspace standalone tipa wasy wasysym xcolor xetex xkeyval
+ENV LIBGS=/usr/lib/x86_64-linux-gnu/libgs.so.9
```

Comment:
This is actually not necessary; setting the `LIBGS` env variable is only required if the ghostscript you want to use is not in a default location or so.

```suggestion
```